data/colors_v2/light.ts

```diff
@@ -0,0 +1,303 @@
+import {alpha, darken, get, lighten, merge} from '../../src/utils'
+import deprecatedVars from './utils/deprecated_vars'
+import lightProductVars from './utils/light_product_vars'
+import lightMarketingVars from './utils/light_marketing_vars'
+
+const vars = {
+  scale: {
+    black: '#1b1f24',
+    white: '#ffffff',
+    gray: [
+      '#fafbfc',
+      '#eaeef2',
+      '#d4dae0',
+      '#b7bfc7',
+      '#949da7',
+      '#707a84',
+      '#57606a',
+      '#424a53',
+      '#32383f',
+      '#24292f'
+    ],
+    blue: [
+      '#ddf4ff',
+      '#b6e3ff',
+      '#80ccff',
+      '#54aeff',
+      '#218bff',
+      '#0969da',
+      '#0550ae',
+      '#033d8b',
+      '#0a3069',
+      '#002155'
+    ],
+    green: [
+      '#dafbe1',
+      '#aceebb',
+      '#6fdd8b',
+      '#4ac26b',
+      '#2da44e',
+      '#1a7f37',
+      '#116329',
+      '#044f1e',
+      '#003d16',
+      '#002d11'
+    ],
+    yellow: [
+      '#fff8c5',
+      '#fae17d',
+      '#eac54f',
+      '#d4a72c',
+      '#bf8700',
+      '#9a6700',
+      '#7d4e00',
+      '#633c01',
+      '#4d2d00',
+      '#3b2300'
+    ],
+    orange: [
+      '#fff1e5',
+      '#ffd8b5',
+      '#ffb77c',
+      '#fb8f44',
+      '#e16f24',
+      '#bc4c00',
+      '#953800',
+      '#762c00',
+      '#5c2200',
+      '#471700'
+    ],
+    red: ['#ffe7e5', '#ffcecb', '#ffaba8', '#ff8182', '#fa4549', '#cf222e', '#a40e26', '#82071e', '#660018', '#4c0014'],
+    purple: [
+      '#fbefff',
+      '#ecd8ff',
+      '#d8b9ff',
+      '#c297ff',
+      '#a475f9',
+      '#8250df',
+      '#6639ba',
+      '#512a97',
+      '#3e1f79',
+      '#2e1461'
+    ],
+    pink: ['#ffeff7', '#ffd3eb', '#ffadda', '#ff80c8', '#e85aad', '#bf3989', '#99286e', '#772057', '#611347', '#4d0336']
+  },
+
+  // Foundations
```

Comment:
Should these global variables be moved to `utils/light_global_vars.ts `? So that all of the themes in the root only contain the color scales.

Or even move the scales to `themes/`:

```
colors_v2/
├─ index.ts
├─ prefix
├─ themes/
│  ├─ light.ts
│  ├─ dark.ts
│  ├─ dark_dimmed.ts
│  ├─ dark_high_contrast.ts
├─ vars/
│  ├─ dark_global.ts
│  ├─ dark_marketing.ts
│  ├─ dark_product.ts
│  ├─ deprecated.ts
│  ├─ light_global.ts
│  ├─ light_marketing.ts
│  ├─ light_product.ts
```

Reply:
I like that. We'll probably also want a `light_component.ts` and `dark_component.ts` for component variables like `btn`

Reply:
Done in https://github.com/primer/primitives/pull/103/commits/b7f94e8cf9c6cf1a94318af9feaf40cb14ac5a4a